src/common/test-jsonfds.c

```diff
@@ -410,7 +410,7 @@ static void
 test_memfd_error_cases (void)
 {
   g_autoptr(GError) error = NULL;
-  g_autofree gchar *content = NULL;
```

Comment:
I don't like this one.

It popped up last time I tested this code and I ignored it because I consider it to be conceptually wrong.  By static typing, this is a string which should be freed!  Consider if someone copies this code as an example...

I wonder if your added assert will be enough to satisfy Coverity....  I'd prefer to keep the `g_autofree` in place.

Reply:
Ack, I put it back, and I'll do another coverity run to validate it. If it complains again, I can add an annotation.

Reply:
No more test-jsonfds.c complaints in the coverity run with this approved PR, so this is good.